src/components/ScoreCard.css

```diff
@@ -1,13 +1,29 @@
+
 .lesson-card-score-card {
     width: 100%;
-}
-
-.lesson-card-score-card-star {
-    width: min(2.8vh, 28px);
+  }
+  
+  .lesson-card-score-card-star {
+    width: min(2.6vh, 26px);
     float: left;
-}
-
-.lesson-card-score-card-middle-star {
-    width: min(2.8vh, 28px);
+    position: relative;
+    
+  }
+  
+  .lesson-card-score-card-middle-star {
+    width: min(2.6vh, 26px);
     float: left;
-}
\ No newline at end of file
+    position: relative;
+    margin: 0 2px;
+  }
+  .star-background-shadow {
+    position: absolute;
+    height: 300%;
+    width: 150%;
```

Comment:
why here height and width more than 100%????

Reply:
okkkk got it but you can use scale here I will accept the PR now next time try to use it